tesla_ble_mqtt/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## 0.0.11
+
+### Changed
+
+- No need to provide MAC addresses anymore for presence detection
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
+- Improved presence detection
+- Improved logging (bashio timestamps, verbosity, colors)
+- Added "debug" entity which sends only one charge amps command: https://github.com/tesla-local-control/tesla_ble_mqtt_core/issues/19
```

Comment:
Is this still valid? I thought we were sending this command twice?! Or is the "debut" entity sending just 1 command to see if it works?!

Reply:
debug entity is to send only once.
So we have the "passe partout" which sends twice if below 5 and the "debug" which sends once only.
For amps below 5a, the debug will not work immediately and will need to be called twice from HA

Reply:
Why is it called "debug"?  Are we troubleshooting this issue?  I feel confused by the word debug.